system/plutono/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart for SAP's CCloud Plutono 7.5.x deployment
 name: plutono
-version: 0.2.6
+version: 0.2.7
 dependencies:
   - name: postgresql
     repository: file://../../common/postgresql
```

Comment:
unrelated, but why isn't this using `https://charts.eu-de-2.cloud.sap`

Reply:
https://github.com/sapcc/helm-charts/blob/641d765d4e7839a272f545625f824aefa0a2c357/common/postgresql/Chart.yaml#L2